frontend/src/scenes/feature-flags/FeatureFlags.tsx

```diff
@@ -122,7 +122,7 @@ export function FeatureFlags(): JSX.Element {
                                         onClick={() => {
                                             deleteWithUndo({
                                                 endpoint: `projects/${currentTeamId}/feature_flags`,
-                                                object: { name: featureFlag.name, id: featureFlag.id },
+                                                object: { name: featureFlag.key, id: featureFlag.id },
```

Comment:
this seems like an unrelated change...

Reply:
It's a definite fix, but I think it fits into the "overhaul" goal. So for feature flags, the `name` field means the description, while the _actual_ "name" of the FF is its `key`. 🤯 Noticed this when testing out the new style